ssds/blobstore/gs.py

```diff
@@ -81,6 +81,8 @@ def delete(self):
         self._get_native_blob().delete()
 
     def copy_from_is_multipart(self, src_blob: "GSBlob") -> bool:
+        # FIXME: Does gs even support multipart? Why would the user_project reflect that?
```

Comment:
It should... the parts are striped and downloaded asynchronously: https://github.com/DataBiosphere/ssds/blob/dc9cab95b0f753a3264d7cdbc07d93aeab824ac3/ssds/blobstore/gs.py#L131

That said, the `user_project` not being included seems odd and is probably a quirk of the object used is my best guess.  Maybe not a great way to flag it if so.

I'm curious to know if you find out conclusively.